src/pudl/extract/ferc714.py

```diff
@@ -2,7 +2,8 @@
 from collections import OrderedDict
 
 import pandas as pd
-from dagster import AssetOut, Output, multi_asset
+from dagster import AssetsDefinition, asset
+from tomlkit import table
```

Comment:
What do you need this for?

Reply:
I don't need this, just came across the hurdle of trying to run some downstream asset and failing that because dagster would refuse to materialize anything that had multi-asset upstream of it unless you manually select all of the constituents of multi-asset.

Reply:
Ah, I haven't realized that the comment specifically refers to this import. Huh, no idea how this came to be.